src/reconcile/rec_visibility.c

```diff
@@ -350,10 +339,8 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
          */
         if (upd->type == WT_UPDATE_TOMBSTONE) {
             upd_select->stop_ts = upd->start_ts;
-            if (upd->txnid != WT_TXN_NONE)
-                upd_select->stop_txn = upd->txnid;
-            if (upd->durable_ts != WT_TS_NONE)
-                tombstone_durable_ts = upd->durable_ts;
+            upd_select->stop_txn = upd->txnid;
```

Comment:
lgtm but I suggest @tetsuo-cpp also have a look at this just to make sure.

Reply:
Exactly the above change has occurred some test failures, I fixed it by removing the following code where it checks for WT_TXN_NONE and set the `upd` to NULL. The TOMBSTONE scenario is already handled in callers of the `__wt_upd_select` function, so I removed the code. @tetsuo-cpp  Can you please verify the changes.

Reply:
Do we still have double tombstones in rollback to stable? I'm not sure if this works with that. In that case, the start timestamp will be set to the second tombstone on the chain but we will still remove the key because the first tombstone to which the stop timestamp is set is globally visible.

Reply:
If that is how it works, it seems confusing and err-prone. Probably we should skip the second tombstone here.

Reply:
Yes, we still have double TOMBSTONE's, but the first one itself is globally visible, so we should go further to check the next TOMBSTONE. The second TOMBSTONE is already skipped. Do you see any problem?

Reply:
Where do we skip the second tombstone? From my understanding, if we have double tombstones, we will set the start time pair to the second tombstones' time pairs, which is the problem I see in a lot of tickets.

Reply:
Double TOMBSTONE's are possible currently only in rollback to stable scenario, and in that the last added TOMBSTONE is a globally visible one. In `__wt_rec_upd_select` function, whenever a globally visible TOMBSTONE is found, the reconciliation logic doesn't move to the next update in the list, so it doesn't take that update time pairs into account.

Reply:
@kommiharibabu @quchenhao 
This timestamp management code has been changing a lot so I haven't been able to keep track of everything.
To my understanding, this looks ok though.